source/develop/troubleshooting-nfs-storage-issues.md

```diff
@@ -166,3 +188,30 @@ In this case it is a good idea to set the tcp window parameters on the NFS serve
 To activate these settings for the running server reload them with
 
        # sysctl -p 
+
+
+## Failure to activate NFS Storage 
+When deploying Self Hosted Engine through cli (ovirt-hosted-engine-setup) or Cockpit web UI, or when adding NFS storage domain to a Standalone manager installation activation of NFS storage fails. The above nfs-check.py will missleadingly succeed.  
+
+### Troubleshooting checklist
+* NFS share mounts successfully briefly but reports failure. You can monitor if NFS share  by running simple while loop on the ovirt node in another terminal. 
+        # while :; do  mount| grep nfs | grep -v rpc_pipefs; done
+* Files are created with correct uid:gid (36:36)
+* Check that sanlock user can  read and write files to the filesystem when mounted manually. 
+* Disable SELinux or AppArmor on the NFS server (as a last resort) doesn't fix the issue.
+
+### Check the logs 
+There are 3 logs to check for storage initialization issues found in /var/log/.
+* vdsm/vdsm.log
+* vdsm/supervdsm.log
+* sanlock.log
```

Comment:
This is very important!

We can add more info about the logs:
- /var/log/vdsm/supervdsm.log - show mount errors
- /var/log/vdsm/vdsm.log - show error when vdsm try modify the mounted export as user vdsm
- /var/log/sanlock.log - show errors when sanlock try to modify the mounted export as user sanlock